rust/src/main.rs

```diff
@@ -21,6 +21,11 @@ fn inner_main(args: &Vec<&str>) -> Result<()> {
 // those, and if there's something we don't know about, invoke the C++
 // main().
 fn main() {
+    if std::env::var("RPMOSTREE_GDB_HOOK").is_ok() {
+        println!("RPMOSTREE_GDB_HOOK detected; stopping...");
+        println!("Attach via gdb using `gdb -p {}`.", std::process::id());
+        unsafe { libc::raise(libc::SIGSTOP) };
```

Comment:
Let's avoid the `unsafe` and use https://docs.rs/nix/0.21.0/nix/sys/signal/fn.raise.html ?

Reply:
Thanks, done!